index.php

```diff
@@ -1,13 +1,49 @@
 <?php
-require 'functions.php';
+include_once 'functions.php';
+?>
 
-echo '<h1>Record Collection</h1>';
-echo '<br>';
-echo '<br>';
-
-$db = getDB();
-$query = $db->prepare('SELECT `Title`, `Rating`, `Released`, `Condition`, `Artist` FROM `records`;');
-$query->execute();
-$records = $query->fetchALL();
-
-displayRecords($records);
\ No newline at end of file
+<html lang="en">
+<!DOCTYPE html>
+<html>
+    <head>
+        <link href="normalize.css" rel="stylesheet" type="text/css">
+        <link href="style.css" rel="stylesheet" type="text/css">
+    </head>
+    <div class="heading"></div><h1>Martin's Record Collection</h1></div>
+        <?php
+        $db = getDB();
+        echo display(extractFromDB($db));?>
+    <div class="container">
+    <h3>Want to add to your collection? Insert your new record below.</h3><br></head>
+<body>
+<form action="add_to_db.php">
+    <label>Artist</label>
+    <input type="text" name="artist">
+    <br><br>
+    <label>Title</label>
+    <input type="text" name="title">
+    <br><br>
+    <label>Year of Release</label>
+    <input name="released" min="1900" max="2100" type="number" step="1">
```

Comment:
interesting date range! 😁  I looked it up, the earlier vinyl record is from 1948, apparently! I wonder if people will still be collecting and producing them in 2101 💭

Reply:
MAYBE I'LL FIND OUT! 🤯 I could live that long!